Testreglar/1.1.1/Nett/nett-1.1.1f.json

```diff
@@ -0,0 +1,129 @@
+{
+	"namn": "Nett-1.1.1f Video eller lydklipp har et tilgjengelig navn med en beskrivende identifikasjon 2023",
+	"id": "nett-1.1.1f",
+	"testlabId": 508,
+	"versjon": "1.0",
+	"type": "Nett",
+	"spraak": "nb",
+	"kravTilSamsvar": "<p>Forhåndsinnspilte video og lydklipp har et tilgjengelig navn med en beskrivende identifikasjon.</p>",
```

Comment:
I spørsmål 3.2 spør vi om video eller lyd er direktesendt. Dersom vi tolker at kravet til tekstalternativ (1.1.1) bare gjelder for forhåndsinnspilt innhold må det går frem i kravet til samsvar.

Reply:
Det står "Forhåndsinnspilte video..." i krav til samsvar. Har du et bedre forslag?

Reply:
Var vell meir ett spørsmål om 1.1.1 har eit slikt unntak?   Er det forskrifta som gjer at vi ender opp med det?

Reply:
Eg har tatt vekk ordet "Forhåndsinnspilt" i krav til samsvar. I tillegg har eg lagt til følgande i steg 3.2. Merk: Suksesskriteriet gjelder etter ordlyden for alle video- og lydklipp. Vi avgrenser likevel testingen mot direktesendt video- og lydklipp. Dette held i massevis.